app/index.tsx

```diff
@@ -26,28 +33,53 @@ export default function Index() {
     );
   }
 
+  const onSignIn = () => {
+    router.navigate("/signin");
+  };
+
+  const onSignUp = () => {
+    router.navigate("/signup");
+  };
+
   return (
     <Screen>
-      <Text
-        style={{
-          color: colorPalette.colors.primary,
-          fontSize: 48,
-          fontWeight: "bold",
-          fontStyle: "italic",
-        }}
-      >
-        DFIA
-      </Text>
-      <Text
-        style={{
-          color: colorPalette.colors.secondary,
-          fontSize: 24,
-          marginBottom: 28,
-        }}
-      >
-        Don't Face It Alone
-      </Text>
-      <SignIn />
+      <Text style={styles.title}>{"DFIA"}</Text>
+      <Text style={styles.subtitle}>{"Don't Face It Alone"}</Text>
+      <TouchableOpacity style={styles.signInBtn} onPress={onSignIn}>
```

Comment:
I think these could be Link components to be simpler
https://docs.expo.dev/router/navigating-pages/#buttons

Reply:
will fix this